common/src/main/java/com/gentics/mesh/auth/provider/MeshJWTAuthProvider.java

```diff
@@ -246,7 +247,8 @@ public String generateAPIToken(com.gentics.mesh.core.data.User user, String toke
 		JsonObject tokenData = new JsonObject()
 			.put(USERID_FIELD_NAME, user.getUuid())
 			.put(API_KEY_TOKEN_CODE_FIELD_NAME, tokenCode);
-		JWTOptions jwtOptions = new JWTOptions().setAlgorithm(options.getAlgorithm());
+		JWTOptions jwtOptions = new JWTOptions().setAlgorithm(options.getAlgorithm())
```

Comment:
Audience and issuer values are mandatory for Vert.x JWT auth implementation, so we have to
* either consider the Mesh settings, while authenticating the token (which may be the breaking change for the existing API tokens of the clients). This option is implemented below.
* or ignore them during the authentication process. Which makes the whole pull request kinda useless.